docs/themerc

```diff
@@ -74,11 +74,20 @@ osd.window-switcher.preview.border.color: #dddda6,#000000,#dddda6
 osd.workspace-switcher.boxes.width: 20
 osd.workspace-switcher.boxes.height: 20
 
-snapping.preview.region.fill: yes
-snapping.preview.edge.fill: yes
-snapping.preview.region.bg.color: #8080b380
-snapping.preview.edge.bg.color: #8080b380
-snapping.preview.region.border.width: 1
-snapping.preview.edge.border.width: 1
-snapping.preview.region.border.color: #dddda6,#000000,#dddda6
-snapping.preview.edge.border.color: #dddda6,#000000,#dddda6
+# Default values for following options change depending on the rendering
+# backend. For software-based renderers, *.fill is "no" and *.stroke is "yes"
+# if not set. For hardware-based renderers, *.fill is "yes" and *.stroke is
+# "no" if not set.
+# Setting *.fill to "yes" for software-based renderer is highly discouraged.
```

Comment:
Suggest something like "may severely impact performance" instead of "is highly discouraged".

Reply:
> I think this is also only an issue when using transparency.

Confirmed in my test. I didn't know that!
Seems like blending in CPU is much heavier operation than just drawing.